source/_components/input_datetime.markdown

```diff
@@ -65,7 +65,7 @@ input_datetime:
 A datetime input entity's state exports several attributes that can be useful in automations and templates.
 
 | Attribute | Description |
-| --------- | ----------- |
+| ... | ... |
```

Comment:
Sorry my fault. Thought about ending each line with `|` until I saw that it wasn't used in the table above, so I deleted the comment. `| --------- | ----------- |` is better here.

Reply:
Done - again 😛